efiction/metadata.py

```diff
@@ -71,17 +71,29 @@ def _convert_authors(self, old_authors=None):
         self.logger.info("Converting authors...")
         current = 0
         total = len(old_authors)
+        query = f"INSERT INTO authors (`id`, `name`, `email`) Values " 
+        query_lines = []
         for old_author in old_authors:
```

Comment:
Could add another comment here explaining what's happening in the loop? (given your commenting elsewhere is excellent)